### DISCUSSION TITLE
Does the alpha of the graphic's fill color affect the interaction ？

### DISCUSSION THREAD

**Discussion by pzppzz** [pixijs/pixijs#9285] [Q&A] [answered]
>
const view = new Container();
view.interactive = true;
const graphics = new Graphics();
view.addChild(graphics);
graphics.beginFill(0xffffff, 1);  // alpha
graphics.lineStyle(1, 0x1890ff, 1);
graphics.drawRect(0, 0, 200, 200);
graphics.endFill();
view.on("mousedown", (evt) => {
   console.log(evt) // it works
})

const view = new Container();
view.interactive = true;
const graphics = new Graphics();
view.addChild(graphics);
graphics.beginFill(0xffffff, 0); // alpha
graphics.lineStyle(1, 0x1890ff, 1);
graphics.drawRect(0, 0, 200, 200);
graphics.endFill();
view.on("mousedown", (evt) => {
   console.log(evt) // it doesn't work
})

**Comment by bigtimebuddy** [comment, marked as answer]
Yes, alpha amount of 0 is ignored from interaction.

**Reply by pzppzz** [reply]
Thanks.
